sdk/main/main.go

```diff
@@ -0,0 +1,155 @@
+package main
+
+import (
+	"fmt"
+	"github.com/conductor-sdk/conductor-go/sdk/client"
+	"github.com/conductor-sdk/conductor-go/sdk/model"
+	"github.com/conductor-sdk/conductor-go/sdk/settings"
+	"github.com/conductor-sdk/conductor-go/sdk/worker"
+)
+
+type Restaurant struct {
+	Restaurant RestaurantData `json:"restaurant"`
+}
+
+type RestaurantData struct {
+	Name  string `json:"name"`
+	Owner Owner  `json:"owner"`
+}
+
+type Owner struct {
+	Name string `json:"name"`
+}
+
+func authSettings() *settings.AuthenticationSettings {
+	key := "api_key_user_03"
+	secret := "api_key_user_03"
+	if key != "" && secret != "" {
+		return settings.NewAuthenticationSettings(
+			key,
+			secret,
+		)
+	}
+
+	return nil
+}
+
+func httpSettings() *settings.HttpSettings {
+	url := "http://localhost:8080/api"
+	return settings.NewHttpSettings(url)
+}
+
+type Data struct {
+	InputKey InputKey `json:inputKey`
+}
+
+type InputKey struct {
+	Headers      Headers `json:"headers"`
+	ReasonPhrase string  `json:"reasonPhrase"`
+	Body         Body    `json:"body"`
+	StatusCode   int     `json:"statusCode"`
+}
+
+type Headers struct {
+	Date                    []string `json:"Date"`
+	ContentType             []string `json:"Content-Type"`
+	ContentLength           []string `json:"Content-Length"`
+	Connection              []string `json:"Connection"`
+	StrictTransportSecurity []string `json:"Strict-Transport-Security"`
+}
+
+type Body struct {
+	RandomString   string                 `json:"randomString"`
+	RandomInt      int                    `json:"randomInt"`
+	HostName       string                 `json:"hostName"`
+	APIRandomDelay string                 `json:"apiRandomDelay"`
+	SleepFor       string                 `json:"sleepFor"`
+	StatusCode     string                 `json:"statusCode"`
+	QueryParams    map[string]interface{} `json:"queryParams"`
+}
+
+func ProcessBody(body *Data) (*Data, error) {
+	if body == nil {
+		fmt.Println("Body is nil")
+		return nil, nil
+	}
+	body.InputKey.Body.HostName = "locally hosting"
+	fmt.Println("returning", body)
+	return body, nil
+}
+
+func ProcessBody2(task *model.Task) (interface{}, error) {
+	fmt.Println("executing", task.TaskId)
```

Comment:
Suggestion: usage of logs instead of print with the correct log level to help in debugging later